Libraries/LibWeb/Crypto/CryptoAlgorithms.cpp

```diff
@@ -3734,7 +3734,7 @@ WebIDL::ExceptionOr<GC::Ref<JS::ArrayBuffer>> ECDH::derive_bits(AlgorithmParams
         auto secret_data = maybe_secret_data.release_value();
         VERIFY(secret_data[0] == 0x04);
 
-        secret = TRY_OR_THROW_OOM(realm.vm(), secret_data.slice(1, secret_data.size() - 1));
+        secret = TRY_OR_THROW_OOM(realm.vm(), secret_data.slice(1, secret_data.size() / 2));
```

Comment:
This could do with a comment explaining why we're slicing the secret data in half.

Reply:
Done